ambry-frontend/src/test/java/com/github/ambry/frontend/FrontendIntegrationTest.java

```diff
@@ -124,30 +124,22 @@
   private static final String HOST_NAME = "localhost";
   private static final String CLUSTER_NAME = "Cluster-name";
   private static boolean enableUndeleteTested = false;
-
```

Comment:
Most of changes in this file are made by auto-reformat in Intellij, I'll try to revert them.